doc/source/whatsnew/v0.21.1.txt

```diff
@@ -91,6 +91,7 @@ I/O
 - Bug in :meth:`DataFrame.to_msgpack` when serializing data of the numpy.bool_ datatype (:issue:`18390`)
 - Bug in :func:`read_json` not decoding when reading line deliminted JSON from S3 (:issue:`17200`)
 - Bug in :func:`pandas.io.json.json_normalize` to avoid modification of ``meta`` (:issue:`18610`)
+- Bug when storing NaN-only categorical columns in hdf5 store (:issue:`18413`)
```

Comment:
Bug when reading ......in a ``:class:`HDFStore` ``

Reply:
ok, done.